Breaking change in ContentType API between 5.3.2 -> 5.4.x

Latest changes in `ContentType` deprecates some properties:

* https://github.com/javalin/javalin/commit/9f4cb1396a12895d19cff6796c53eb6383403db5

One of them is marked as `ERROR`:

https://github.com/javalin/javalin/blob/9f4cb1396a12895d19cff6796c53eb6383403db5/javalin/src/main/java/io/javalin/http/ContentType.kt#L80

And that leads to the hard compilation error in Kotlin, so it's a breaking change between two minor versions:

```
16 actionable tasks: 3 executed, 13 up-to-date
e: File.kt:55:29 Using 'APPLICATION_JS' is an error. use TEXT_JS instead.

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':xxx:compileKotlin'.
> A failure occurred while executing org.jetbrains.kotlin.compilerRunner.GradleCompilerRunnerWithWorkers$GradleKotlinCompilerWorkAction
   > Compilation error. See log for more details
````

--- a/javalin/src/main/java/io/javalin/http/ContentType.kt
+++ b/javalin/src/main/java/io/javalin/http/ContentType.kt
@@ -77,7 +77,7 @@ enum class ContentType(
     @Deprecated(
         message = "use TEXT_JS instead.",
         replaceWith = ReplaceWith("io.javalin.http.ContentType.TEXT_JS"),
-        level = DeprecationLevel.ERROR
+        level = DeprecationLevel.WARNING
     )
     APPLICATION_JS("application/javascript", true, "application/javascript"),
     APPLICATION_JSON("application/json", true, "json"),

